Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -265,6 +265,20 @@ def check_plist(prefix, plist)
     EOS
   end
 
+  PYTHON_SYMLINK_ALLOWED_FORMULA = "python@3.8"
```

Comment:
Do we have this defined somewhere else? Or can we infer it from the keg_only status?

Reply:
> Do we have this defined somewhere else?

Not that I can find

> Or can we infer it from the keg_only status?

Probably. This would be ideal to avoid needing to update the blocklist for every new python version.

Reply:
Yeah, that was exactly what I was worried about.

Reply:
Done! I think it clarifies the check anyway 👍